CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.15.1 (July 2020)
+- Android 10 support
+- Update network libraries to more recent versions, OkHttp + Dav4Android
+- Bug fixes, including:
+  + Fix some authentication problems
+  + Add support for version.hide server setting
```

Comment:
How does this change affect users? Couldn't we explain this in a more user-friendly way?